pywt/_extensions/_dwt.pyx

```diff
@@ -98,9 +98,9 @@ cpdef dwt_axis(np.ndarray data, Wavelet wavelet, MODE mode, unsigned int axis=0)
     return (cA, cD)
 
 
-cpdef idwt_single(data_t[::1] cA, data_t[::1] cD, Wavelet wavelet, MODE mode):
-    cdef data_t[::1] rec
+cpdef idwt_single(np.ndarray cA, np.ndarray cD, Wavelet wavelet, MODE mode):
```

Comment:
Huh, very odd. Leave it in then, I'll investigate it later. Thanks for checking it out though.